src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  width: calc((7 * $column-width) + (6 * $gap));
```

Comment:
The width property should be calculated using the formula `width: calc(100% / 7 * $days-in-week - $gap);` where `$days-in-week` is the number of days in a week. This ensures that the width is responsive to the number of days and the gap size.